script.transmission/README.md

```diff
@@ -0,0 +1,26 @@
+Transmission-Kodi
```

Comment:
transmission-kodi violates our trademark policy...

Please rename it to just Transmission, Transmission Client, or "Transmission for Kodi or whatever that doesn't cause confusion as the source.

Reply:
Transmission Control Panel sounds right?

Reply:
Sure :)